src/components/shared/Warning.css

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I essentially reused the styles that I used for the service worker reload notice.